src/ocean/util/serialize/contiguous/package_.d

```diff
@@ -734,10 +734,10 @@ unittest
 {
     static struct CS
     {
-        cstring s;
+        mstring s;
```

Comment:
Is this change unrelated to "Temporarily disable test that check const fields" then?